source/_patterns/02-organisms/components/aside.mustache

```diff
@@ -0,0 +1,32 @@
+<aside class="aside">
```

Comment:
Rename the component `content-aside.mustache`. I did think that `article-aside` might work but I can see the use for this on other content pages.

Reply:
The component is now named 'content-aside'